.github/workflows/cd-manual.yml

```diff
@@ -0,0 +1,26 @@
+name: CloudFlare Pages Deploy
+on: 
+  workflow_dispatch:
+
+jobs:
+  deploy:
+      runs-on: ubuntu-latest
+      permissions:
+        contents: read
+        deployments: write
+      steps:
+          - uses: actions/checkout@v3
+          - uses: actions/setup-node@v3
+            with:
+              cache: yarn
+              node-version: '16'
+          - run: yarn install --frozen-lockfile
```

Comment:
You should run `corepack enable` before and use --immutable i stead of --frozen-lockfile here